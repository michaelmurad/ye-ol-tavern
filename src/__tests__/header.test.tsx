import React from "react"
import { render } from "@testing-library/react"
import Header from "../components/header"

it("renders correctly", () => {
  const { container } = render(<Header siteTitle="hello" />)
  expect(container.firstChild.textContent).toBe("hello")
  expect(container.firstChild.classList).toMatchSnapshot()
})
