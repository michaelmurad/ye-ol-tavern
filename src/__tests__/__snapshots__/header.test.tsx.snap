// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `"font-fancy p-2 text-5xl text-black bg-primary"`;
